ctapipe/image/image_processor.py

```diff
@@ -143,8 +148,8 @@ def _parameterize_image(
                     container_class=PeakTimeStatisticsContainer,
                 )
             else:
-                timing = TimingParametersContainer()
-                peak_time_statistics = PeakTimeStatisticsContainer()
+                timing = DEFAULT_TIMING_PARAMETERS
+                peak_time_statistics = DEFAULT_PEAKTIME_STATISTICS
```

Comment:
Codecov is complaining that these 2 lines are not covered by unit-testing

Reply:
That could be done by testing the stage1 tool on a file with true images. But let's leave that to another PR